pages/thc/[processRef]/[...slug].tsx

```diff
@@ -116,7 +118,16 @@ const ProcessPage: NextPage = () => {
         steps={innerSteps}
         manageStepTransitions={false}
         provideDatabase={false}
+        onIncompleteStep={(missingValues?: string[]): void => {
+          if (missingValues) {
+            // need to get real label of this, e.g. What type of ID, rather than e.g. `id-type` from component?
+            console.log(missingValues);
+            setMissingValues(missingValues);
+            throw new Error("required field");
```

Comment:
I don't think we should be treating this as an error. The application is working fine. It's the user who performed an error. I see that this is probably a workaround for `onSubmit` being called regardless. I'll fix that on `remultiform` to return `false` from `onSubmit` when submission should be interrupted.

Reply:
See https://github.com/LBHackney-IT/remultiform/pull/985